opm/simulators/flow/Main.hpp

```diff
@@ -325,77 +326,89 @@ class Main
 
 #if HAVE_DAMARIS
         enableDamarisOutput_ = EWOMS_GET_PARAM(PreTypeTag, bool, EnableDamarisOutput);
+        // Reset to false as we cannot use Damaris if there is only one rank.
+        if ((enableDamarisOutput_ == true) && (EclGenericVanguard::comm().size() == 1)) {
+            std::string msg ;
+            msg = "\nUse of Damaris (command line argument --enable-damaris-output=true) has been dissabled for run with only one rank.\n" ;
+            OpmLog::info(msg);
```

Comment:
Make this a warning() instead of info() message.

Reply:
This test and change of: *enableDamarisOutput_ = false* does not propagate through to eclwriter.hh, which again tests for the value of the command line argument (using *EWOMS_GET_PARAM(PreTypeTag, bool, EnableDamarisOutput)*).
  
This causes some calls to the Damaris API to be called via the EclWriter class and tests for failure blemish the output of the OPM-Flow run.
  
Is there a way to set the EWOMS parameter? something like a EWOMS_SET_PARAM() ?

Reply:
I think there is no way to set it. We might add it, but I will try a different fix. For now I'll run Jenkins on this and merge if green. Thanks for making the changes I requested!

Reply:
Ok for that. 

I'll try to get another pull request with the larger set of changes available soon - I am trying to split the work into two, however I am running into extra issues, so it may be a bigger, single request.